.gitlab/kitchen_testing/windows.yml

```diff
@@ -109,26 +166,49 @@
     - .kitchen_os_windows
   needs: ["deploy_windows_testing-a7"]
 
+.kitchen_scenario_windows_a7_ec2:
+  extends:
+    - .kitchen_agent_a7
+    - .kitchen_os_windows_ec2
+  needs: ["deploy_windows_testing-a7"]
+
 # Kitchen: final test matrix (test types * scenarios)
 # ----------------------------------------------
 
-kitchen_windows_installer_npm_install_scenarios-a7:
+##
+## temporarily remove; being run on EC2 instead until Azure issue ironed out
```

Comment:
Can you specify the issue here? So we can check if it's fixed and move this back to Azure. If this doesn't have to ever be moved back to Azure, then remove the commented code.